reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt

```diff
@@ -238,12 +239,16 @@ class AntennaAttributionDocumentReporter : Reporter {
         return outputFiles
     }
 
-    private fun createCopyrightStatement(resolvedLicense: ResolvedLicenseInfo) =
-        resolvedLicense.flatMapTo(sortedSetOf()) { license ->
+    private fun createCopyrightStatement(resolvedLicense: ResolvedLicenseInfo): String {
+        val copyrights = resolvedLicense.filterExcluded().flatMapTo(sortedSetOf()) { license ->
             license.locations.flatMap { location ->
                 location.copyrights.map { it.statement }
             }
-        }.joinToString("\n")
+        }
```

Comment:
I would recommend to replace `license.locations.flatMap { ...}` with `license.getCopyrights(omitExcluded = true)`.

Reply:
Using `license.getCopyrights()` which also omits the excluded copyrights, therefore I removed the `filterExcluded()`